Common/UnitDefinitions/RatioChangeRate.json

```diff
@@ -0,0 +1,32 @@
+﻿{
+  "Name": "RatioChangeRate",
+  "BaseUnit": "PercentPerSecond",
+  "XmlDoc": "The quantity of percent flowing per unit of time.",
+  "Units": [
+    {
+      "SingularName": "PercentPerSecond",
+      "PluralName": "PercentsPerSecond",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "%/s" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "DecimalFractionPerSecond",
+      "PluralName": "DecimalFractionsPerSecond",
+      "FromUnitToBaseFunc": "x*1e2",
+      "FromBaseToUnitFunc": "x/1e2",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "" ]
```

Comment:
Hmm.. should this not have an abbreviation? Not sure what, but would perhaps `1/s` work? Just something to signify the time dimensions we clearly have.